io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl

```diff
@@ -0,0 +1,303 @@
+   samm:preferredName "Identifier Characteristic"@en;
+   samm:description "Characteristic for identifiers."@en;
+   samm:dataType xsd:string.
+
+:MaterialSetCharacteristic a samm-c:Set;
+   samm:preferredName "Material Set Characteristic"@en;
+   samm:description "Characteristic for a set of materials."@en;
+   samm:dataType :MaterialEntity.
+
+:PropertySetCharacteristic a samm-c:Set;
+   samm:preferredName "Property Set Characteristic"@en;
+   samm:description "Characteristic for a set of property representations."@en;
+   samm:dataType :PropertyEntity.
+
+:HierarchyElementSetCharacteristic a samm-c:Set;
+   samm:preferredName "Hierarchy Element Set Characteristic"@en;
+   samm:description "Characteristic for a set of hierarchy elements."@en;
+   samm:dataType :HierarchyElementIdEntity.
+
+:HierarchyElementIdCharacteristic a samm:Characteristic;
+   samm:preferredName "Hierarchy Element Id Characteristic"@en;
+   samm:description "Characteristic for an identifier of a hierarcy element."@en;
+   samm:dataType xsd:string.
+
+:CapabilityConstraintEntity a samm:Entity;
+   samm:preferredName "Capability Constraint Entity"@en;
+   samm:description "Element containing the samm properties of a capability constraint."@en;
+   samm:properties ( [ samm:property :capabilityConstraintProperties; samm:optional true ] [ samm:property :refersToMaterial; samm:optional true ] ).
+
+:SemanticReferenceEntity a samm:Entity;
+   samm:preferredName "Semantic Reference Entity"@en;
+   samm:description "Element containing the SAMM properties of a semantic reference."@en;
+   samm:properties ( :sementicReferenceId ).
+
+:MaterialEntity a samm:Entity;
+   samm:extends :ElementAbstractEntity;
+   samm:preferredName "Material Entity"@en;
+   samm:description "Element containing the SAMM properties of a material."@en;
+   samm:properties ( [ samm:property :belongsToMaterialFamilies; samm:optional true ] ).
+
+:PropertyEntity a samm:Entity;
+   samm:preferredName "Property Entity"@en;
+   samm:description "Element containing the SAMM properties of a property."@en;
+   samm:properties ( :propertyLabel [ samm:property :semanticReferences; samm:optional true ] [ samm:property :propertyValue; samm:optional true ] ).
+
+:HierarchyElementIdEntity a samm:Entity;
+   samm:preferredName "Hierarchy Element Id Entity "@en;
+   samm:description "Element containing the SAMM properties of a hierarchy element."@en;
+   samm:properties ( :hierarchyElementId ).
+
+:capabilityConstraintProperties a samm:Property;
+   samm:preferredName "Capability Constraint Properties"@en;
+   samm:description "Set of constraints restricting the capability."@en;
+   samm:characteristic :PropertySetCharacteristic.
+
+:refersToMaterial a samm:Property;
+   samm:preferredName "Refers to Material"@en;
+   samm:description "A specific material the capability constraint can refer to."@en;
+   samm:characteristic :MaterialCharacteristic.
+
+:sementicReferenceId a samm:Property;

```

Comment:
correct name should be `semanticReferenceId`. typo in property name. you can consider renaming it if you want.